.github/workflows/packagist.yml

```diff
@@ -20,19 +20,37 @@ jobs:
         with:
           go-version: "1.24"
 
-      - name: Build shared library
-        run: make build
+      - name: Install cross-compilation tools
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y gcc-aarch64-linux-gnu
+
+      - name: Build shared libraries for all platforms
+        run: |
+          # Build Linux AMD64 (native)
+          make build-linux-amd64
+
+          # Build Linux ARM64 (cross-compile)
+          make build-linux-arm64
+
+          # Build macOS binaries (cross-compile)
+          # Note: macOS builds may have limitations in Linux CI
+          make build-darwin-amd64 || echo "macOS AMD64 build skipped"
+          make build-darwin-arm64 || echo "macOS ARM64 build skipped"
```

Comment:
We should fail if the build fails, otherwise we'd have no library for those architectures and break stuff for whoever will be using them on a new relase